src/cook-web/src/components/dnd-kit-sortable-tree/ui/simple/SimpleTreeItemWrapper.tsx

```
@@ -1,93 +1,93 @@
-) => React.ReactElement;
+/* eslint-disable react/display-name */
+/* eslint-disable @typescript-eslint/no-empty-object-type */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import clsx from 'clsx';
+import React, { forwardRef } from 'react';
+import type { TreeItemComponentProps } from '../../types';
+import './SimpleTreeItemWrapper.css';
+
+export const SimpleTreeItemWrapper = forwardRef<
+  HTMLDivElement,
+  React.PropsWithChildren<TreeItemComponentProps<{}>>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)